data/bactrian.txt

```diff
@@ -97,20 +97,23 @@ mission "Bactrian Unlock [0]: Prime"
 	source Prime
 	to offer
 		has "Bactrian Unlock [0]: active"
+		not "found star queen"
 	on offer
 		conversation
 			`The Star Queen is not in this system, so you attempt to find somebody who may have some information. After visiting several shops, you find a merchant who was clearing cargo through customs as the Star Queen departed. As she shares her experience, you can't help but be glad the Deep Security officer is not with you, as to hear this tale unfold would surely have shaken him further.`
- 			`	According to the merchant, a large pirate fleet jumped in, eviscerated the escorts, and escaped north towards Rigel with the hostage Star Queen. You can only hope the trail is not already cold.`
+			`	According to the merchant, a large pirate fleet jumped in, eviscerated the escorts, and escaped north towards Rigel with the hostage Star Queen. You can only hope the trail is not already cold.`
 				decline
 
 mission "Bactrian Unlock [0]: Farpoint"
 	landing
 	source Farpoint
 	to offer
 		has "Bactrian Unlock [0]: active"
+		not "found star queen"
 	on offer
 		conversation
-			`After speaking with a Navy Officer for a few minutes, you find out that the Star Queen came through here over a week ago, escorted by several large pirate ships. The Navy did not want to risk the lives of those on the Star Queen by engaging the pirate fleet, so they let it continue north from here.`			`	The spaceport clearly shows preparations for a rescue mission, but due to the destruction of a number of Navy capital ships by the Star Queen's captors in the days preceding its abduction, they have been ordered to wait for additional ammunition and reinforcements.`
+			`After speaking with a Navy Officer for a few minutes, you find out that the Star Queen came through here over a week ago, escorted by several large pirate ships. The Navy did not want to risk the lives of those on the Star Queen by engaging the pirate fleet, so they let it continue north from here.`
```

Comment:
Given the description of Farpoint (it's *only* a a naval base), Navy can be dropped too.

-> After speaking with an officer.....